src/pages/index.js

```diff
@@ -1,25 +1,26 @@
 import Link from "next/link";
-import Image from "next/image";
 
 export default function Home() {
   return (
     <div>
       <main className="home">
-        <h1>Accessibility Cheatsheets for Components
-        </h1>
-        <p className="home__header-description">
-          Intro copy goes here, such as: Build accessible components 
-          more accurately and in less time with these easy-to-read 
-          cheatsheets based on WCAG and ARIA best practices. 
-          Check back often as we’re adding new components frequently! 
-        </p>
-        <Image id="home__header-image" src="/home-header-image.svg" width="100%" height="100%" />
+        <div className="home-top">
+          <h1>Accessibility Cheatsheets for Components
```

Comment:
For consistency, I'd make sure that opening/closing tags are consistently on the same line or a different line than what they're encapsulating.